disperser/apiserver/server.go

```diff
@@ -507,13 +532,15 @@ func (s *DispersalServer) GetBlobStatus(ctx context.Context, req *pb.BlobStatusR
 
 	requestID := req.GetRequestId()
 	if len(requestID) == 0 {
+		s.metrics.HandleInvalidArgRpcRequest("GetBlobStatus")
 		s.metrics.HandleInvalidArgRequest("GetBlobStatus")
```

Comment:
These should be removed later: quorum tracking doesnt' apply to GetBlobStatus

Reply:
Why can't we remove it now?

Reply:
It can be a sanity check for the new metric with new v.s. old comparison